languages/python/sqlalchemy-oso/tests/conftest.py

```diff
@@ -98,7 +99,10 @@ def db_uri():
 
 @pytest.fixture
 def engine(db_uri):
-    engine = create_engine(db_uri)
+    if AT_LEAST_SQLALCHEMY_VERSION_1_4:
+        engine = create_engine(db_uri, enable_from_linting=False)
```

Comment:
Why `enable_from_linting=False`?

Reply:
🤷 I didn't feel like updating our test fixtures when I set that flag

here's the relevant changelog entry: https://docs.sqlalchemy.org/en/14/changelog/migration_14.html#change-4737

Reply:
I hadn't really thought through whether I'll want to keep the flag or fix the fixtures once I do a clean-up pass on the initial code. Thoughts welcome!

Reply:
Replaced flag with a `try...except` block

Reply:
Could you add a comment about why `enable_from_linting` is necessary and open a clean up task in notion about this? I don't think we should need to disable this in our tests.